L1Trigger/TrackTrigger/plugins/TTStubBuilder.h

```diff
@@ -79,6 +99,12 @@ TTStubBuilder< T >::TTStubBuilder( const edm::ParameterSet& iConfig )
 {
   clustersToken = consumes< edmNew::DetSetVector< TTCluster< T > > >(iConfig.getParameter< edm::InputTag >( "TTClusters" ));
   ForbidMultipleStubs = iConfig.getParameter< bool >( "OnlyOnePerInputCluster" );
+  applyFE             = iConfig.getParameter< bool >( "FEineffs" );
```

Comment:
put all of these initializations in constructor initialization list

Reply:
Not done